hedera-node/src/main/java/com/hedera/services/store/tokens/unique/UniqueTokenStore.java

```diff
@@ -129,7 +128,7 @@ public ResponseCodeEnum wipe(final AccountID aId, final TokenID tId, final long
 
 	private boolean checkProvisional(List<Pair<MerkleUniqueTokenId, MerkleUniqueToken>> provisionalUniqueTokens) {
```

Comment:
Based on our weekly sync yesterday -> we decided that we will accept `TokenMint`s with the same `metadata` meaning that we can remove this check